media-plugins/vdr-xineliboutput/vdr-xineliboutput-2.3.0.ebuild

```diff
@@ -51,11 +60,12 @@ DEPEND="${COMMON_DEPEND}
 		)
 	)"
 RDEPEND="${COMMON_DEPEND}"
-BDEPEND="virtual/pkgconfig"
+BDEPEND="acct-user/vdr
```

Comment:
Doesn't that need also be in ```RDEPEND```?

Reply:
Hello @ConiKost ,
when this package is build, only with USE=vdr the vdr user+group is needed. And in this moment, then there is media-video/vdr in DEPEND and RDEPEND and media-video/vdr itself depends on acct-usr/vdr. So I see no problem here.